src/LandPlanner.cpp

```diff
@@ -72,8 +83,11 @@ bool LandPlanner::prepareForTakeover(const ros::Time& time)
         return false;
     }
 
-    requested_height_ = transform.transform.translation.z;
+    requested_height_ = transform.transform.translation.z; 
 
+    // height determined by the ratio of landing accelerations
+    cushion_height_ = std::min(0.5 * (std::pow(descend_rate_,2)/cushion_acceleration_), 
```

Comment:
We need to look at the offset between the landing gear and the center of lift to calculate this properly as revealed by testing.